src/main/java/analyzer/exercises/needforspeed/AvoidConditionalLogic.java

```diff
@@ -0,0 +1,15 @@
+package analyzer.exercises.needforspeed;
+
+import analyzer.Comment;
+
+public class AvoidConditionalLogic extends Comment {
```

Comment:
Please follow the convention used in the other exercises, where a bit of Javadoc is used to link to the corresponding Markdown template in the website-copy repository.

Reply:
Done!